src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -733,7 +735,25 @@ public class SimpleEvents {
 					.requiredPlugins("Paper 1.16+");
 		}
 
-		Skript.registerEvent("Player Pickup Arrow", SimpleEvent.class, PlayerPickupArrowEvent.class, "[player] (pick[ing| ]up [an] arrow|arrow pick[ing| ]up)")
+		Skript.registerEvent("Entity Breed", SimpleEvent.class, EntityBreedEvent.class, "[entity] breed[ing]")
```

Comment:
I think it could be a good idea for these events to support triggering conditions (e.g. `on pig breed`) using a dedicated event class

Reply:
hmm, not a bad idea, I'll need to relook at this PR again soon, with all the changes recently there's bound to be one or two conflicts in addition to all the changes regarding conventions